### ISSUE TITLE
lost connection

### ISSUE THREAD

**Issue by HausamSee2024** [lkempf/casambi-bt-hass#132]
Hi there, 
since two weeks i cannot connect to Casambi anymore. I did not change anything. Casambi says they did not change accessibility. Do you have an idea what might have went wrong?

Thanks a lot, Michael
